src/features/user/UserProfile/components/Settings.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
```

Comment:
You should use an editor which can automatically nicely format our source code as in the file before.